doc/api/policy.md

```diff
@@ -0,0 +1,42 @@
+# Policies
+
+<!--introduced_in=TODO-->
+<!-- type=misc -->
+
+> Stability: 0 - Experimental
+
+<!-- name=policy -->
+
+Node.js contains experimental support for creating policies on loading code.
+
+## Enabling
+
+<!-- type=misc -->
+
+The `--experimental-policy` flag can be used to enable features for policies when loading modules.
+
+Once this has been set, all modules must conform to a policy file passed to the flag
+
+```sh
+node --experimental-policy policy.json app.js
+```
+
+## Features
+
+### Integrity Checks
+
+Policy files must use integrity checks with Subresource Integrity strings compatible with the [integrity attribute as present in browsers](https://www.w3.org/TR/SRI/#the-integrity-attribute) associated with absolute URLs.
+
+And example policy file that would let you load a file `checked.js` would be:
+
+```json
+{
+  "resources": {
+    "file:///path/to/checked.js": {
+      "integrity": "sha384-SggXRQHwCG8g+DktYYzxkXRIkTiEYWBHqev0xnpCxYlqMBufKZHAHQM3/boDaI/0"
+    }
+  }
+}
+```
+
+In order to generate integrity strings, a script such as `printf "sha384-$(cat checked.js | openssl dgst -sha384 -binary | base64)"` can be used.
```

Comment:
This is not really friendly for people on Windows. We should provide a tool on npm to automate this.

Reply:
I can agree to that but a generalized manifest tool seems something that we aren't ready to make a design for while we still are working out the design of manifests themselves.

Reply:
This is not really a blocker, but something that we should put into the roadmap for this to get out of experimental.